Date: Wed, 18 Aug 2004 21:18:19 +0200
From: Andreas Gruenbacher <agruen@suse.de>
Subject: Resolve duplicate/conflicting MODULE_LICENSE tags

* drivers/infiniband/ulp/srp/ib_srp.ko is missing the MODULE_LICENSE tag,
  but is Dual BSD/GPL according to source code comments.

* fs/affs/affs.ko has MODULE_LICENSE("GPL") twice

* drivers/infiniband/hw/mellanox-hca/hh/mod_hh.ko has MODULE_LICENSE("GPL")
  twice

* drivers/scsi/pcmcia/fdomain_cs.ko has both MODULE_LICENSE("GPL") and
  MODULE_LICENSE("Dual MPL/GPL"). The common denominator seems to be
  MODULE_LICENSE("GPL").

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

Index: linux-2.6.8/drivers/infiniband/hw/mellanox-hca/hh/hh.c
===================================================================
--- linux-2.6.8.orig/drivers/infiniband/hw/mellanox-hca/hh/hh.c
+++ linux-2.6.8/drivers/infiniband/hw/mellanox-hca/hh/hh.c
@@ -198,7 +198,6 @@ HH_ret_t HHIF_dummy()
 #ifdef MT_KERNEL
 
 
-MODULE_LICENSE("GPL");
 int init_hh_driver(void)
 {
   HH_hca_dev_t*  dev = devBegin;
Index: linux-2.6.8/drivers/infiniband/hw/mellanox-hca/hh/hhklsyms.c
===================================================================
--- linux-2.6.8.orig/drivers/infiniband/hw/mellanox-hca/hh/hhklsyms.c
+++ linux-2.6.8/drivers/infiniband/hw/mellanox-hca/hh/hhklsyms.c
@@ -23,7 +23,7 @@
 #include <hh.h>
 #include <hh_common.h>
 
-MODULE_LICENSE("GPL");
+MODULE_LICENSE("Dual BSD/GPL");
 
 
 EXPORT_SYMBOL(HH_add_hca_dev);
Index: linux-2.6.8/fs/affs/inode.c
===================================================================
--- linux-2.6.8.orig/fs/affs/inode.c
+++ linux-2.6.8/fs/affs/inode.c
@@ -427,4 +427,3 @@ err:
 	affs_unlock_link(inode);
 	goto done;
 }
-MODULE_LICENSE("GPL");
Index: linux-2.6.8/drivers/infiniband/ulp/srp/srp_host.c
===================================================================
--- linux-2.6.8.orig/drivers/infiniband/ulp/srp/srp_host.c
+++ linux-2.6.8/drivers/infiniband/ulp/srp/srp_host.c
@@ -23,6 +23,7 @@
 
 #include "srp_host.h"
 
+MODULE_LICENSE("Dual BSD/GPL");
 MODULE_PARM ( dlid_conf, "i");
 MODULE_PARM_DESC (dlid_conf, "dlid_conf (nonzero value indicates that dlid is being specified in the conf file)");
 tUINT32 dlid_conf = 0;
Index: linux-2.6.8/drivers/scsi/pcmcia/fdomain_stub.c
===================================================================
--- linux-2.6.8.orig/drivers/scsi/pcmcia/fdomain_stub.c
+++ linux-2.6.8/drivers/scsi/pcmcia/fdomain_stub.c
@@ -59,7 +59,6 @@
 
 MODULE_AUTHOR("David Hinds <dahinds@users.sourceforge.net>");
 MODULE_DESCRIPTION("Future Domain PCMCIA SCSI driver");
-MODULE_LICENSE("Dual MPL/GPL");
 
 /* Bit map of interrupts to choose from */
 static int irq_mask = 0xdeb8;
